[2024-03-27T03:48:45.357+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Process_W3_Data.check_directories manual__2024-03-27T03:48:42.050043+00:00 [queued]>
[2024-03-27T03:48:45.372+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Process_W3_Data.check_directories manual__2024-03-27T03:48:42.050043+00:00 [queued]>
[2024-03-27T03:48:45.373+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-27T03:48:45.396+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): check_directories> on 2024-03-27 03:48:42.050043+00:00
[2024-03-27T03:48:45.407+0000] {standard_task_runner.py:60} INFO - Started process 479 to run task
[2024-03-27T03:48:45.414+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Process_W3_Data', 'check_directories', 'manual__2024-03-27T03:48:42.050043+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/w3c.py', '--cfg-path', '/tmp/tmp2vgutoab']
[2024-03-27T03:48:45.421+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask check_directories
[2024-03-27T03:48:45.517+0000] {task_command.py:423} INFO - Running <TaskInstance: Process_W3_Data.check_directories manual__2024-03-27T03:48:42.050043+00:00 [running]> on host 94418fe46b42
[2024-03-27T03:48:45.639+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Process_W3_Data' AIRFLOW_CTX_TASK_ID='check_directories' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T03:48:42.050043+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-27T03:48:42.050043+00:00'
[2024-03-27T03:48:45.643+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-27T03:48:45.644+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '[ -d /home/***/gcs/data/Raw/ ] && echo "Raw directory exists" || echo "Raw directory does not exist"']
[2024-03-27T03:48:45.668+0000] {subprocess.py:86} INFO - Output:
[2024-03-27T03:48:45.681+0000] {subprocess.py:93} INFO - Raw directory does not exist
[2024-03-27T03:48:45.683+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-27T03:48:45.724+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Process_W3_Data, task_id=check_directories, execution_date=20240327T034842, start_date=20240327T034845, end_date=20240327T034845
[2024-03-27T03:48:45.787+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-27T03:48:45.823+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
